app/views/home/maintainers.md

```diff
@@ -0,0 +1,45 @@
+> Audience: "developers", meaning anyone who has skills that are relevant to software development, that identify as a "maintainer" regularly contribute to an open source project.
+
+> What are you asking me to do?
+
+Take one day this week to make your project easier to use and more welcoming to new contributors.
+
+> Why should I care?
+
+You could use help
+
+> What are the reasons I should care about getting more contributors for my open source project?
```

Comment:
I think the question here isn't "why should I care about getting more contributors" (we should assume they're on this page bc they want them), but "why should I spend all this extra time on a project that is already a PITA"

AKA how does cleaning up my project attract more contributors?